### PR TITLE
Fixes #2081. Introduced flexbox styles for report buttons.

### LINKED ISSUE #2081: Report as buttons not aligned when signed in
The form buttons for reporting anonymous or on Github are not aligned once I sign in.

screen shot 2018-02-15 at 4 00 04 pm

### PR DESCRIPTION
**Pull request by Regaddi**
As mentioned in #2081.

r? @zoepage

### CODE REVIEW

File: webcompat/static/css/src/form.css [hunk 1]
```diff
@@ -27,6 +27,21 @@
...
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap-reverse;
+  justify-content: space-between;
+  margin-left: -10px;
+  margin-right: -10px;
+  max-width: none;
+}
+
+.form-button.form-button-multiple button {
```
Comment (zoepage) [inline webcompat/static/css/src/form.css]: As we want to separate 100% the semantics and the styles, we want to styles for html elements. Could you please change `button` to `.button`?

File: webcompat/static/css/src/form.css [hunk 2]
```diff
@@ -27,6 +27,21 @@
   margin: 0 0 20px;
 }
 
+.form-button.form-button-multiple {
```
Comment (zoepage) [inline webcompat/static/css/src/form.css]: I guess, we can remove `.form-button` due to specificity.

File: webcompat/static/css/src/form.css (hunk 1 above)
Comment (zoepage) [inline webcompat/static/css/src/form.css]: I guess, we can remove `.form-button` due to specificity.

**Comment by zoepage** [comment]
Thanks! There are just small changes :)

Also, we want to keep the order of the buttons. (Sorry, if we misunderstood each other).

screen shot 2018-03-13 at 4 35 46 pm
screen shot 2018-03-13 at 4 35 54 pm

We'd need to switch the order of the buttons in the smaller resolution. 


Thanks!

**Comment by Regaddi** [comment]
There you go @zoepage

**Comment by zoepage** [comment]
LGTM!

**Pull request merged**